src/index.js

```diff
@@ -0,0 +1,48 @@
+const env = require('./env');
+const migrations = require('./migrations');
+const refreshMatViews = require('./refresh-materialized-views');
+const log = require('loglevel');
+
+log.setDefaultLevel('info');
+
+if (!env.validateEnv()) {
+  process.exit(1);
+  return; // stop execution in tests
+}
+
+if (process.argv.length > 2 && process.argv[2] === '--usage') {
+  log.info(`
+  USAGE
+  
+  export POSTGRESQL_URL=<PostgreSQL db URL>
+  export RAPIDPRO_URL=<RapidPro instance URL>  
+  export RAPIDPRO_AUTH=<RapidPro authentication token>
+  rapidpro2pg
+  `);
+  process.exit(0);
+}
+
+const syncEndpoints = async() => {
+  const endpoints = [
+    'contacts',
+    'messages',
+    'runs',
+    'flows',
+  ];
+
+  for (const endpoint of endpoints) {
+    await (require(`./endpoints/${endpoint}.js`)).sync();
```

Comment:
We can add here a log before and after stating that "XXX task started..." and then "XXX task ended..." so users see the progress.

Same before and after `migrations.run()` and `refreshMatViews.run()`.

Reply:
I'll add the logs in the utils files and keep the index file clean.